helix-term/src/ui/editor.rs

```diff
@@ -940,14 +941,14 @@ impl EditorView {
 }
 
 impl Component for EditorView {
-    fn handle_event(&mut self, event: Event, cx: &mut Context) -> EventResult {
-        let mut cxt = commands::Context {
-            editor: cx.editor,
+    fn handle_event(&mut self, event: Event, context: &mut crate::compositor::Context) -> EventResult {
```

Comment:
This was changed to clarify this was the compositor context, and to use the same naming conventions as the rest of the codebase.